src/Microsoft.Azure.SignalR.Protocols/ConnectionMessage.cs

```diff
@@ -82,6 +82,74 @@ public OpenConnectionMessage(string connectionId, Claim[] claims, IDictionary<st
         public string Protocol { get; set; }
     }
 
+    /// <summary>
+    /// Close connections in a hub
+    /// </summary>
+    public class CloseConnectionsWithAckMessage : ExtensibleServiceMessage, IAckableMessage, IMessageWithTracingId
```

Comment:
Shall we introduce an abstract message with `AckId`, `TracingId`, and `ExcludedList` fields in it?

Reply:
updated